session-v7.go

```diff
@@ -32,14 +32,14 @@ import (
 	"github.com/minio/minio/pkg/quick"
 )
 
-func migrateSessionV5ToV6() {
+func migrateSessionV6ToV7() {
```

Comment:
yes, I saw you did that with config migration, however in session, migration does nothing more than deleting old session files, this code  will clean all versions inferior to 7